client/pages/Login.tsx

```diff
@@ -2,61 +2,14 @@ import React, {useState, useEffect} from 'react';
 import { KeyboardAvoidingView, StyleSheet, Text, TextInput, TouchableOpacity, View } from 'react-native';
 import { useTheme } from '@react-navigation/native';
 import { useNavigation } from '@react-navigation/core';
+import AsyncStorage from '@react-native-async-storage/async-storage';
```

Comment:
Very cool library! I've been grabbing _uid_  from the _auth_ object but this is maybe an easier way to have it accessible everywhere.

Reply:
Sounds neat. I will need to spend some time learning about how to use this.